public/data/Sulagna-Dutta-Roy.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Sulagna Dutta Roy",
+  "type": "personal",
+  "bio": "Frontend Developer",
+  "avatar": "https://github.com/sulagna-dutta-roy.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/sulagna-dutta-roy",
+      "icon": "github"
+    },
+    {
+      "name":"Connect with me on Linkedin",

```

Comment:
I recommend capitalizing the `i` here. What do you think?
```suggestion
      "name":"Connect with me on LinkedIn",
```

Reply:
Done the changes. Kindly check it